### PR TITLE
Update receivers README

### PR DESCRIPTION
**Pull request by flands**
**Documentation:** Update receivers README

### CODE REVIEW

File: receiver/README.md [hunk 1]
```diff
@@ -68,25 +71,27 @@ The full list of settings exposed for this receiver are documented [here](https:
 with detailed sample configurations [here](https://github.com/open-telemetry/opentelemetry-collector/blob/master/receiver/opencensusreceiver/testdata/config.yaml).
 
 ### Communicating over TLS
-This receiver supports communication using Transport Layer Security (TLS). TLS can be configured by specifying a `tls-crendentials` object in the receiver configuration for receivers that support it.   
+This receiver supports communication using Transport Layer Security (TLS). TLS
+can be configured by specifying a `tls-crendentials` object in the receiver
```
Comment (pjanotti) [inline receiver/README.md]:
```suggestion
can be configured by specifying a `tls_crendentials` object in the receiver
```

File: receiver/README.md [hunk 2]
```diff
@@ -223,14 +231,27 @@ receivers:
 ## <a name="vmmetrics"></a>VM Metrics Receiver
 **Only metrics are supported.**
 
-<Add more information - I'm lonely.>
+Collects metrics from the host operating system. This is applicable when the
+OpenTelemetry Collector is running as an agent.
+```yaml
+receivers:
+  vmmetrics:
+    scrape_interval: 10
```
Comment (pjanotti) [inline receiver/README.md]:
```suggestion
    scrape_interval: 10s
```

File: receiver/README.md [hunk 3]
```diff
@@ -223,14 +231,27 @@ receivers:
...
+    mount_point: "/proc"
+    # process_mount_point: "/data/proc" # Should only be used for Daemon Set within a container.
+```
 
 ## <a name="zipkin"></a>Zipkin Receiver
 **Only traces are supported.**
 
-This receiver receives spans from Zipkin (V1 and V2) HTTP uploads and translates them into the internal span types that are then sent to the collector/exporters.
+This receiver receives spans from Zipkin (V1 and V2) HTTP uploads and
+translates them into the internal span types that are then sent to the
```
Comment (pjanotti) [inline receiver/README.md]:
```suggestion
translates them into the internal span types that are then pushed into trace pipelines according to the configuration.
```

File: receiver/README.md [hunk 4]
```diff
@@ -129,7 +134,9 @@ It is possible to configure the protocols on different ports, refer to
 examples.
 
 ### Communicating over TLS
-This receiver supports communication using Transport Layer Security (TLS), but only using the gRPC protocol. It can be configured by specifying a `tls-crendentials` object in the gRPC receiver configuration.   
+This receiver supports communication using Transport Layer Security (TLS), but
+only using the gRPC protocol. It can be configured by specifying a
+`tls-crendentials` object in the gRPC receiver configuration.
```
Comment (tigrannajaryan) [inline receiver/README.md]: Should this be `tls_crendentials`?

File: receiver/README.md (hunk 1 above)
Comment (flands) [reply receiver/README.md]: err also a type: credentials

**Comment by flands** [comment]
@pjanotti @tigrannajaryan -- thanks, fixed

File: receiver/README.md [hunk 5]
```diff
@@ -223,14 +231,27 @@ receivers:
...
-<Add more information - I'm lonely.>
+Collects metrics from the host operating system. This is applicable when the
+OpenTelemetry Collector is running as an agent.
+```yaml
+receivers:
+  vmmetrics:
+    scrape_interval: 10s
+    metric_prefix: "testmetric"
+    mount_point: "/proc"
+    # process_mount_point: "/data/proc" # Should only be used for Daemon Set within a container.
```
Comment (owais) [inline receiver/README.md]: > Should only be used for Daemon Set within a container.

This isn't very clear.

Comment (flands) [reply receiver/README.md]: Updated slightly -- not an expert on this, appears to be related to differences between deployment models.

**Pull request merged**